content/libraries/go.mdx

```diff
@@ -3,12 +3,108 @@ title: Go
 description: Official Top.gg Go library
 ---
 
-# Go Library
+## Links
+[Documentation](https://pkg.go.dev/github.com/DiscordBotList/go-dbl)
 
-This is our official Go Library for top.gg, if you have any issues/bugs please submit an issue on our github.
+[Go Package](https://pkg.go.dev/github.com/DiscordBotList/go-dbl)
 
-- [GitHub Link](https://github.com/top-gg/go-sdk)
+[Github](https://github.com/top-gg/go-sdk)
 
-## Usage
+## Installation
 
-Please refer to the github docs
+```
+go get -u github.com/top-gg/go-dbl
+```
+
+## Posting bot stats
+```go
+package main
+
+import (
+	"log"
+
+	"github.com/top-gg/go-dbl"
+)
+
+func main() {
+	dblClient, err := dbl.NewClient("token")
+	if err != nil {
+		log.Fatalf("Error creating new Discord Bot List client: %s", err)
+	}
+
+	err = dblClient.PostBotStats("botID", &dbl.BotStatsPayload{
```

Comment:
Is there an example or link to the sharded version too?

Reply:
I couldn't find one when I was looking